actions/update.go

```diff
@@ -34,7 +34,7 @@ func containerFilter(names []string) container.Filter {
 // used to start those containers have been updated. If a change is detected in
 // any of the images, the associated containers are stopped and restarted with
 // the new image.
-func Update(client container.Client, names []string, cleanup bool, noRestart bool) error {
+func Update(client container.Client, names []string, cleanup bool, noRestart bool, noUpdate bool) error {
```

Comment:
What would you think about adding a new action `MonitorOnly`? As a result we wouldn't need to add the if statement in the update action.